fvtest/omrGtestGlue/omrTest.h

```diff
@@ -31,24 +31,9 @@ static int iconv_init_static_variable = iconv_initialization();
 #endif /* defined(J9ZOS390) && !defined(OMR_EBCDIC) */
 
 
```

Comment:
Please also delete this blank line (we don't need more than one blank line in a row).